src/FileCache.php

```diff
@@ -450,4 +453,27 @@ private function iterableToArray(iterable $iterable): array
         /** @psalm-suppress RedundantCast */
         return $iterable instanceof Traversable ? iterator_to_array($iterable) : (array) $iterable;
     }
+
+    /**
+     * Check if error was because of file was already deleted by another process on high load
+     */
+    private function isLastErrorNotSafe(mixed $result): bool
```

Comment:
```suggestion
    private function isLastErrorSafe(bool $result): bool
```

And use above, for example:

```php
return  $this->isLastErrorSafe(@touch($file, $expiration));
```